File: Java/백준/Bronze/29790. 임스의 메이플컵/임스의 메이플컵.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int a = in.nextInt();
        int b = in.nextInt();
        if(n>=1000 && (a>=8000 || b>=260)){
            System.out.println("Very Good");
        }
        else{
            System.out.println(n>=1000?"Good":"Bad");
        }
    }
}

File: C#/백준/Bronze/29790. 임스의 메이플컵/임스의 메이플컵.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            if(a[0]>=1000 && (a[1]>=8000 || a[2]>=260)){
                Console.WriteLine("Very Good");
            }
            else if(a[0]>=1000){
                Console.WriteLine("Good");
            }
            else{
                Console.WriteLine("Bad");
            }
        }
    }
}